### DISCUSSION TITLE
Wiki updates

### DISCUSSION THREAD

**Discussion by Juul03** [Matthew-Tangk/project-tech-A2#17] [General]
# Team Rules wiki
"Code Standards" en " Concept & Roles" zijn bijgewerkt in de wiki. Laat maar weten of er nog aanvullingen zijn!

**Comment by Juul03** [comment]
Check vooral effies de routes, dat we het daar allemaal mee eens zijn en op 1 lijn zitten, staan onder het kopje Hierarchie bij "Concept & Roles"

**Reply by Serenoiia** [reply]
Jaa ziet er goed uit zo! Moet ik het alvast de bestaande aanpassen in de routes zodat alles klopt?

**Reply by Serenoiia** [reply]
Zal ik anders de styling en flow verplaatsen naar de "Design & Styling" page. Dan kan ik daar ook nog even tekst aanvullen en een Adobe XD prototype van maken :)

**Reply by Juul03** [reply]
Klinkt goed! Dan staat alles netjes gesorteerd

**Reply by Serenoiia** [reply]
Topp! ik heb het aangepast

**Comment by Serenoiia** [comment]
@Juul03 @rachelou @Matthew-Tangk FF een reminder.  Vergeet niet om je stukje in te vullen bij de [Database structure](https://github.com/Matthew-Tangk/project-tech-A2/discussions/database),  [Process](https://github.com/Matthew-Tangk/project-tech-A2/discussions/process), Je eigen Feature en de [Performance](https://github.com/Matthew-Tangk/project-tech-A2/discussions/performance). Morgen vullen we wel de gezamenlijke stukjes in  die nog ontbreken op school! :)

**Reply by Juul03** [reply]
Yes! Thanks voor de reminder. Gisteren al mijn eigen feature ingevuld, zag dat jij dat ookal had gedaan, zag er goed en volledig uit
